test/multi-runtime-services.test.ts

```diff
@@ -0,0 +1,103 @@
+import * as assert from 'assert';
+import * as path from 'path';
+import { cleanOpenRuntimes, launchAndConnect } from './multi-runtime-utils';
+import { delayPromise } from './delay-promise';
+import * as sinon from 'sinon';
+import * as fs from 'fs';
+
+describe ('Multi Runtime Services', () => {
+    beforeEach(async () => {
+        await cleanOpenRuntimes();
+    });
+
+    after(async () => {
+        await cleanOpenRuntimes();
+    });
+
+    // tslint:disable-next-line
+    describe('Multi Runtime with External Provider', function () {
+
+        it('Should work in multi runtime with an external Provider', function(done: any) {
+            // tslint:disable-next-line no-invalid-this
+            this.timeout(8000);
+            const appConfig = JSON.parse(fs.readFileSync(path.resolve('test/app.json')).toString());
```

Comment:
this could be done once, similar to: https://github.com/HadoukenIO/js-adapter/blob/develop/test/multi-runtime-utils.ts#L13